pkg/storage/backend-storage/backend-storage.go

```diff
@@ -118,3 +118,20 @@ func CreateIfNeeded(vmi *corev1.VirtualMachineInstance, clusterConfig *virtconfi
 
 	return err
 }
+
+func IsPVCPending(vmi *corev1.VirtualMachineInstance, client kubecli.KubevirtClient) (bool, error) {
+	if !IsBackendStorageNeededForVMI(&vmi.Spec) {
+		return false, nil
+	}
+
+	pvc, err := client.CoreV1().PersistentVolumeClaims(vmi.Namespace).Get(context.Background(), PVCForVMI(vmi), metav1.GetOptions{})
+	if err != nil {
+		return false, err
+	}
+
+	if pvc.Status.Phase != v1.ClaimBound {
+		return true, nil
+	}
+
+	return false, nil
+}
```

Comment:
This is used from the vmi controller which has a pvcInformer. How about using it?

Reply:
Well if the PVC just got created, the informer might not have been updated yet, triggering an error.
In general, it seems like a better idea to get the latest status instead of risking failing the whole `sync()` on an outdated phase.